### ISSUE TITLE
Delete the smaller image automatically?

### ISSUE THREAD

**Issue by MatPoot** [ermig1979/AntiDupl#111]
Currently there is a process in the application where the application will automatically select recommended actions for certain duplicates and you can command it to carry out those recommendation actions. 

I can't seem to find a way for the application to auto-mark image duplication in a group and delete all but the largest image file in terms of resolution or file size. Is there a way to do this?

**Comment by heinrichI** [comment]
I have tried implement configirable actions in WPF version https://github.com/ermig1979/AntiDupl/issues/57#issuecomment-487377786:
![image](https://user-images.githubusercontent.com/7752557/98368339-9ee1c200-2050-11eb-8fb4-ab14df48f3b4.png)
Is this what you want? If so, I can improve that version.

**Comment by MatPoot** [comment]
Yes, that would work. Automatically recommend the smaller image for deletion and keep the larger one. 

If there was a way to do it with file-types as well that would be a nice extra, but not necessary ( if same resolution keep the PNG instead of the JPEG  for example, or you could simply just flag the smaller ( in terms to file size ) for deletion.

**Comment by heinrichI** [comment]
Specify more precisely what is missing or what is not working.

**Comment by MatPoot** [comment]
![image](https://user-images.githubusercontent.com/45025651/98539538-c18afb00-2249-11eb-9c0b-7abe1aee8ccb.png)

To add - 
Delete smaller picture

**Comment by heinrichI** [comment]
This picture is from the WinForm version. Nobody is going to develop it. I can work with WPF version.
But in essence: adding another delete button with a smaller size will not solve the problem. This is one of many actions. What if I need to rename a small resolution picture to a name like a large one? Or move? But only if the picture with a small resolution is in this folder, and the resolution is more than 10% and the clarity is not less. For such cases, too, a button?
And so delete with less, as far as I remember, you can now configure it in the WPF version.

**Comment by NOBB2333** [comment]
actually not all the people need this function, but it relly sometimes we need  
my job sometimes use this to find the problem who make fake pic,  
and with python to  output a info, that's a risk in some place
